lib/options/storage.js

```diff
@@ -12,43 +12,44 @@ options.forEach((item) => {
 
 export const get = key => store[key];
 
-export const set = (key, value) => {
+export const set = async (key, value) => {
   const data = {
     [key]: value,
   };
 
-  // Firefox 51.0.1 doesn't have storage.sync, so wrap the property access in a
-  // Promise so that the exception can be handled.
-  return new Promise(resolve => resolve(chromep.storage.sync.set(data)))
-    .catch(() => chromep.storage.local.set(data));
+  try {
+    return await chromep.storage.sync.set(data);
+  } catch (err) {
+    return chromep.storage.local.set(data);
```

Comment:
Yes. I had originally put `await` in there, but eslint complained:

    /Users/josephfrazier/workspace/octolinker/browser-extension/lib/options/storage.js
      23:12  error  Redundant use of `await` on a return value  no-return-await

This makes sense because `async` functions always return a promise, so you don't have to `await` the return value. The reason that `await` is used inside the `try` block is so that any errors with `chromep.storage.sync.set(data)` will be caught inside this function, rather than being exposed to the function caller.